### PR TITLE
Update shinyapps.io workflow to copy schematic config file to schemat…

### PR DESCRIPTION
**Pull request by afwillia**
…ic directory. Otherwise schematic will crash.

### CODE REVIEW

**Comment by rrchai** [comment]
@afwillia Do you know what's the error for the crash?

**Comment by afwillia** [comment]
@rrchai 

> FileNotFoundError: [Errno 2] No such file or directory: 'tests/data/example.model.jsonld'

**Comment by rrchai** [comment]
@afwillia Thanks. The DCA should not use the `config.yml` file from the "schematic" folder (unless it's due to the latest changes from backend). It's okay to add the changes you have in this PR to fix the issue, but I suspect there are something wrong to track down the reason the error appeared as well. Can you please tell me a bit more which branch/instance you encountered this error?

**Comment by afwillia** [comment]
@rrchai interesting - this [fork of DCA for INCLUDE](https://github.com/include-dcc/data_curator) has the error

**Comment by rrchai** [comment]
@afwillia Looks like it's missing the [metadata_model.py](https://github.com/Sage-Bionetworks/data_curator/blob/d427d9521eb696f7c6e5863f4233a61f53843cd1/functions/metadata_model.py#L5), where it defines where the config.yml should read to config `schematic`

**Comment by rrchai** [comment]
@afwillia It is interesting... I just tested on this repo with the latest schematic. It works fine without copying `schematic_config.yml` to `schematic` folder. I am looking into it.

**Comment by afwillia** [comment]
@rrchai that's so weird. I wonder if this is a workflow issue or something with the code. At any rate, I really appreciate your eyes on this.

**Comment by rrchai** [comment]
@afwillia No problem. The reason is that `metadata_model.py` was reading the `schematic/config.yml` before you pull the changes from upstream today - [here](https://github.com/include-dcc/data_curator/commit/f8bc834cd3326455f98aca694edcf188cd4db2cd#diff-05b657849eae74b6489a714b0ebcf835b08ede13a0bb1c5b1794e66cfdbb7d3a) to see the file changes. 

Can you please try to deploy again without without copying schematic_config.yml to schematic folder?

File: .github/workflows/shinyapps_deploy.yml [hunk 1]
```diff
@@ -57,11 +57,14 @@ jobs:
           if [[ $GITHUB_REF_NAME == v*.*.* ]] || [[ $GITHUB_REF_NAME == main ]]; then
             echo Installing pypi version of schematic
             git clone --single-branch --branch main https://github.com/Sage-Bionetworks/schematic.git
+            cp schematic_config.yml schematic/config.yml
+            cd schematic
```
Comment (rrchai) [inline .github/workflows/shinyapps_deploy.yml:61]: `pip install` should not be affected the current working directory, while `poetry install` requires to be in the "schematic" folder.

**Review by rrchai — CHANGES_REQUESTED** [review]

**Comment by afwillia** [comment]
@rrchai thanks so much - that fixed the issue.

**Pull request merged**